### PR TITLE
Release 0.0.5 changes

### PR DESCRIPTION
**Pull request by neel-astro**
Changes required while testing out the release process manually

### CODE REVIEW

**Comment by sunkickr** [comment]
@neel-astro do we need to merge this PR?

File: godownloader.sh [hunk 1]
```diff
@@ -294,7 +294,7 @@ github_release() {
   version=$2
   if [[ "$version" =~ ^v[0-9]+\.[0-9]+$ ]]; then
```
Comment (isaulv) [inline godownloader.sh:295]: change the default from `bin` to `/usr/local/bin`. This would have parity with the legacy astro cli.

**Comment by isaulv** [comment]
can you change the name of the checksum file from `astro-cli` to `astro`?

File: godownloader.sh (hunk 1 above)
Comment (neel-astro) [reply godownloader.sh:295]: the setting of `/usr/local/bin` in the current CLI is done in `install.astronomer.io` bash script logic (Reference: https://github.com/astronomer/install.astronomer.io/blob/main/index.html#L11)

This is mostly an auto-generated file, so the idea is to have as minimal changes as required.

Comment (isaulv) [reply godownloader.sh:295]: Can you find out how to change the default?

**Review by sunkickr — APPROVED** [review]

**Pull request closed without merging**